Automattic/jetpack
# My Jetpack: Add JITM container

This PR adds a container for JITMs to the My Jetpack screen. It looks like this: 

![image](https://user-images.githubusercontent.com/7129409/153609809-79954f9f-6480-441d-8372-87b30319bca9.png)

#### Changes proposed in this Pull Request:

Adds JITM container to My Jetpack screen. 
Adds custom style rules in JITM css so it's compatible with the `<Col>` used in My Jetpack. 

#### Jetpack product discussion


Design convo: p1HpG7-eMU-p2
PT for showing JITM in standalone: pbNhbs-38V-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






- Connect to your sandbox with D73609-code to get the JITM.
- Visit the My Jetpack admin page
- Visit My Jetpack without the JITM enabled to ensure no side effects of adding the empty container/col.
